sdk/java-sdk/src/main/scala/kalix/javasdk/impl/timer/TimerSchedulerImpl.scala

```diff
@@ -58,7 +58,7 @@ private[kalix] final class TimerSchedulerImpl(anySupport: AnySupport, system: Ac
 
   def cancel(name: String): CompletionStage[Done] = {
     val timerServiceClient = GrpcClients(system).getProxyGrpcClient(classOf[TimerService])
-    timerServiceClient.remove(Name(name)).asJava.thenApply(_ => Done)
+    timerServiceClient.remove(StringValue(name)).asJava.thenApply(_ => Done)
```

Comment:
I have changed it long time ago in the proxy, but somehow I still had this in my local build and didn't see the error. 🤷